### PR TITLE
Added search bar for project on start page

### PR DESCRIPTION
**Pull request by denektenina**
Now you will no longer see all projects in sidebar. Instead, you are prompted to select project before you can go further.

image

image

image

### CODE REVIEW

File: frontend/src/Components/SideMenu/SideMenu.tsx [hunk 1]
```diff
@@ -1,17 +1,24 @@
 import React from 'react'
+import { useNavigate, useParams } from 'react-router-dom'
 import styled from 'styled-components'
+import { chevron_left } from '@equinor/eds-icons'
+
 import ProjectMenu from './ProjectMenu'
+import { Divider, Icon, Typography } from '@equinor/eds-core-react'
```
Comment (maiaht) [inline frontend/src/Components/SideMenu/SideMenu.tsx:7]: Move line 7 to line 5

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -20,7 +19,6 @@ function App() {
         <div className="App" style={{ display: 'flex', flexDirection: 'column', height: '100vh', width: '100vw' }}>
             <Header />
             <div style={{ display: 'flex', flexDirection: 'row', flexGrow: 1 }}>
-                <SideMenu />
                 <Outlet />
             </div>
```
Comment (maiaht) [inline frontend/src/App.tsx:23]: Is it necessary to have a `div` around `<Outlet />`?

File: frontend/src/Components/SideMenu/SideMenu.tsx (hunk 1 above)
Comment (denektenina) [reply frontend/src/Components/SideMenu/SideMenu.tsx:7]: Moved

File: frontend/src/App.tsx (hunk 2 above)
Comment (denektenina) [reply frontend/src/App.tsx:23]: Yes, it's necessary for the styling

**Review by maiaht — APPROVED** [review]

**Pull request merged**